web_accounts_list.json

```diff
@@ -2087,6 +2142,17 @@
         "category" : "music",
         "valid" : true
       },
+      {
+        "name" : "skeb",
+        "check_uri" : "https://skeb.jp/@{account}",
+        "account_existence_code" : "200",
+        "account_existence_string" : "Sent Requests",
+        "account_missing_code" : "404",
+        "account_missing_string" : "Not Found",
+        "known_accounts" : ["eipuru_"],

```

Comment:
Please add a second account for the `known_accounts`

Reply:
I also made the checker go through the API now, so it should be faster now!